app/models/iam_user_identity.rb

```diff
@@ -70,7 +72,8 @@ def set_expire
   # @return [String] UUID that is unique to this user
   #
   def uuid
-    Digest::UUID.uuid_v5(@iam_sec_id, @icn)
+    Digest::UUID.uuid_v5(IAM_NAMESPACE, @icn)
+    # Digest::UUID.uuid_v5(@iam_sec_id, @icn)
```

Comment:
We prob don't need this comment, we can just have users look up the git history for the history here I'd say

Reply:
yeah it was just there to toggle between the two while i'm debugging the spec failures. It'll be removed